app/main.py

```diff
@@ -2,0 +1,22 @@
+import os
+from typing import Any
+
+
+def move_file(command: str) -> Any:
+    parts = command.split()
```

Comment:
Consider using unpacking for the `parts` list: `_, word_in, word_out = parts`. This makes the code cleaner and more readable.